sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/TimeoutTests.java

```diff
@@ -0,0 +1,107 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.storage.blob;
+
+import com.azure.core.test.utils.MockTokenCredential;
+import com.azure.core.util.Context;
+import com.azure.storage.blob.models.ListBlobContainersOptions;
+import com.azure.storage.blob.models.ListBlobsOptions;
+import com.azure.storage.blob.options.FindBlobsOptions;
+import com.azure.storage.common.test.shared.extensions.RequiredServiceVersion;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.parallel.Isolated;
+
+import java.time.Duration;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@Isolated
+public class TimeoutTests {
+    /*
```

Comment:
As part of the description, we should also add a note on why we are mocking out the HTTP layer in addition to isolating the tests. Mainly it is not immediately obvious why we are not sending a request to the API.